nautobot/core/tests/test_views.py

```diff
@@ -58,3 +59,44 @@ def test_subdirectory_routes(self):
             set_script_prefix(original_prefix)
 
         self.assertEqual(get_script_prefix(), original_prefix)
+
+
+class NavRestrictedUI(TestCase):
+    def make_request(self):
+        response = self.client.get(reverse("home"))
+        response_body = response.content.decode(response.charset).replace("\n", "")
+        return re.compile(r"<nav.*Installed Plugins.*</nav>").search(response_body)
```

Comment:
So we're looking for "Installed Plugins" anywhere inside the entire `nav` menu block? Can we be any narrower/more specific than that?